lighthouse-core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const UsesHTTP2Audit = require('../../../audits/dobetterweb/uses-http2.js');
+const assert = require('assert');
+
+/* eslint-env mocha */
+
+describe('Resources are fetched over http/2', () => {
```

Comment:
since you're computing directly against the network records, it would be nice to test on an actual copy of the network records. There's one using h2 push in core/test/fixtures/networkRecords-h2push.json. I thought there was one more, but you can generate one via Lighthouse if you pass in the `--save-artifacts` flag when running against some site.